docs/src/SUMMARY.md

```diff
@@ -1,39 +1,49 @@
+<!-- markdownlint-disable MD025 -->
 # Summary
 
 [The Fuel Indexer](./the-fuel-indexer.md)
 
-- [Quickstart](./quickstart/index.md)
-- [Getting Started](./getting-started/index.md)
-  - [Installing Rust](./getting-started/installing-rust.md)
-  - [System dependencies](./getting-started/system-dependencies.md)
-  - [Application dependencies](./getting-started/application-dependencies/index.md)
-    - [Database](./getting-started/application-dependencies/database.md)
-    - [`fuelup`](./getting-started/application-dependencies/fuelup.md)
-    - [`sqlx`](./getting-started/application-dependencies/sqlx.md)
-    - [`wasm-snip`](./getting-started/application-dependencies/wasm-snip.md)
-  - [Configuration](./getting-started/configuration.md)
-  - [A Fuel Indexer Project](./getting-started/fuel-indexer-project.md)
-- [Examples](./examples/index.md)
-  - [Hello World](./examples/hello-world.md)
-  - [Block Explorer](./examples/block-explorer.md)
-- [Plugins](./plugins/index.md)
-  - [`forc index`](./plugins/forc-index.md)
-- [Components](./components/index.md)
-  - [Assets](./components/assets/index.md)
-    - [Manifest](./components/assets/manifest.md)
-    - [Schema](./components/assets/schema.md)
-    - [Module](components/assets/module.md)
-  - [Database](./components/database/index.md)
-    - [Data Types](./components/database/types.md)
-      - [ID Types](./components/database/ids.md)
-    - [Foreign Keys](./components/database/foreign-keys.md)
-  - [GraphQL](./components/graphql/index.md)
-    - [Schema](./components/assets/schema.md)
-    - [API Server](./components/graphql/api-server.md)
-    - [Types](./components/database/types.md)
-    - [Directives](./components/graphql/directives.md)
-  - [Indices](./components/indices/index.md)
-    - [Registering Indices](./components/indices/registration.md)
+[Quickstart](./quickstart/index.md)
+
+# User Guide
```

Comment:
I like the "User Guide" and "Reference Guide" breakdown (feel free to add the inspiration for that to the PR description - for others who might be curious where this breakdown comes from)